include/zephyr/bluetooth/audio/pbp.h

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Define a Doxygen group and add to Bluetooth group (see other Bluetooth profiles for reference, including how to spell out the acronym in the group name and brief description)

Reply:
Done ,thank you

Reply:
> Done ,thank you

mmm no? Maybe this was overriden in a later update or something?